mpa/cls/stage.py

```diff
@@ -204,18 +204,20 @@ def configure_task(cfg, training, model_meta=None, **kwargs):
 
                 if not cfg.model.get('multilabel', False) and not cfg.model.get('hierarchical', False):
                     efficient_mode = cfg['task_adapt'].get('efficient_mode', True)
-                    gamma = 2 if efficient_mode else 3
                     sampler_type = 'balanced'
 
                     if len(set(model_classes) & set(dst_classes)) == 0 or set(model_classes) == set(dst_classes):
                         cfg.model.head.loss = dict(type='CrossEntropyLoss', loss_weight=1.0)
                     else:
                         cfg.model.head.loss = ConfigDict(
-                            type='SoftmaxFocalLoss',
-                            loss_weight=1.0,
-                            gamma=gamma,
-                            reduction='none',
+                            type='IBLoss',
```

Comment:
Wouldn't it be appropriate to change it in the model recipe rather than the stage?
Also, is IB loss only used for class-incremental?

Reply:
Since IB loss is used for c-il, I put it in stage.py. Please refer to my comment of Vlad's review above.

Reply:
I think the problem is that loss-overriding doesn't happen now (because hard-declaration in stage.py), so shouldn't this be fixed?

Reply:
Why do you think loss-overriding is not happening? I checked IB loss works in incremental setting.

Reply:
What I was saying was that adding a loss in the recipe should reflect the actual training, but currently it doesn't. So, it seems to be possible to solve this and add IB loss to the recipe.